### PR TITLE
[GSoC] Function for plotting shear stress added

### PR DESCRIPTION
**Pull request by Psycho-Pirate**
#### Brief description of what is fixed or changed

Function for plotting Shear stress is added in Beam class and Beam3D class.

#### Release Notes

* physics.continuum_mechanics 
   * Added ability to plot shear stress in Beam class and Beam3D class

### CODE REVIEW

**Comment by Psycho-Pirate** [comment]
@ishanaj

File: sympy/physics/continuum_mechanics/beam.py [hunk 1]
```diff
@@ -3166,3 +3230,105 @@ def plot_loading_results(self, dir='x', subs=None):
...
+        for sym in shear_stress[dir_num].atoms(Symbol):
+            if sym != self.variable and sym not in subs:
+                raise ValueError('Value of %s was not passed.' %sym)
+        if self.length in subs:
+            length = subs[self.length]
+        else:
+            length = self.length
+
+        return plot(shear_stress[dir_num].subs(subs), (self.variable, 0, length), show = False, title='Shear stress along %c direction'%dir,
+                xlabel=r'$\mathrm{X}$', ylabel=r'$\sigma(%c)$'%dir, line_color=color)
```
Comment (ishanaj) [inline sympy/physics/continuum_mechanics/beam.py:3262]: Why is the symbol for shear stress in 3D beams `sigma` but `tau` for 2D beams?

Comment (Psycho-Pirate) [reply sympy/physics/continuum_mechanics/beam.py:3262]: I forgot to change it

**Review by ishanaj — APPROVED** [review]

**Comment by ishanaj** [comment]
Looks good to me! Merging..!

**Pull request merged**